libyul/optimiser/README.md

```diff
@@ -74,6 +74,22 @@ and F is a list of function definitions such that no function contains a functio
 
 The benefit of this stage is that we always know where the list of function begins.
 
+### For Loop Condition Into Body
+
+This transformation moves the iteration check part of a for-loop into loop body.
```

Comment:
`iteration check condition` instead of `iteration check part`?

Reply:
Then I would fully switch to `iteration condition`